R/GroupSequentialDesign.R

```diff
@@ -35,54 +35,28 @@ GSDesign <- function(n1, c1f, c1e, n2_pivots, c2_pivots, x1_norm_pivots, weights
         stop("x1_norm_pivots must be in [-1, 1], is scaled automatically")
     if (any(weights <= 0))
         stop("weights must be positive")
-    new("GSDesign", n1 = n1, c1f = c1f, c1e = c1e, n2_pivots = n2_pivots,
-        c2_pivots = c2_pivots, x1_norm_pivots = x1_norm_pivots, weights = weights)
+    tunable <- logical(8) # initialize to all false
+    tunable[1:5] <- TRUE
+    names(tunable) <- c("n1", "c1f", "c1e", "n2_pivots", "c2_pivots", "x1_norm_pivots", "weights", "tunable")
+        new("GSDesign", n1 = n1, c1f = c1f, c1e = c1e, n2_pivots = n2_pivots,
+        c2_pivots = c2_pivots, x1_norm_pivots = x1_norm_pivots, weights = weights,
+        tunable = tunable, rounded = FALSE)
```

Comment:
I get the point of introducing a new filed 'rounded' - but don't you think this a bit over the top? I'd just implemented it as n2(x1, round = FALSE) in the corresponding method - keeping changes and complexity low.
If you want to keep it, I am fine with it but it is a bit over-complicated ... (if I am not missing something)

Ah, okay - maybe I see. Making this a design property means we do not need to alter anything in the score evaluation? Good point